README.md

```diff
@@ -267,6 +268,25 @@ This method will clear the local storage containing the current connected user a
 
 If you pass a callback, the page won't be reload. As a result, you can directly call the `signIn()` method again if you want. The `init()` and `discover()` methods don't need to be called again.
 
+### Restore tokens
+
+If you refresh your page, or after a restart of the browser, it could happens that the JS library loose the tokens of the user. So after, you can't call the identity method, or the signOut by example.
+
+To prevent this, if you have saved the access token and id token in one way or another, in your session system by ex, or in app device, you can restore the "connected" status of the libray calling the method restoreTokens with tokens in arguments.
+
+```javascript
+AsmodeeNet.restoreTokens('eY[...]', 'eJ5[...]');
+```
+
+After this call, the JS Library as the same behavior as if it just do the sign in call.
+
+The good way it's to do the call to restoreTokens, if you are in the case you have to do it, just after the `init` / `discover` calls.
+
+Parameters of the method `restoreTokens`:
+* the first one is the **access token** as string
+* the seconde one is the **id token** as string
+* you can pass a third parameter, boolean, true by default. If it's trus, the library, after validate tokens, call the identity() methode with callback configured in the init method for signin callback (callback_signin_success and callback_signin_error)
```

Comment:
If it's true, the library, after it validates the tokens, calls the identity() method with the callback configured in the init method for signin callback (callback_signin_success and callback_signin_error).